.github/scripts/strategy-matrix/generate.py

```diff
@@ -238,21 +240,145 @@ def generate_strategy_matrix(all: bool, config: Config) -> list:
         # Add the configuration to the list, with the most unique fields first,
         # so that they are easier to identify in the GitHub Actions UI, as long
         # names get truncated.
-        configurations.append(
-            {
-                "config_name": config_name,
-                "cmake_args": cmake_args,
-                "cmake_target": cmake_target,
-                "build_only": build_only,
-                "build_type": build_type,
-                "os": os,
-                "architecture": architecture,
-            }
-        )
+        # Add Address and Thread (both coupled with UB) sanitizers when the distro is bookworm.
```

Comment:
```suggestion
        # Add Address and Thread (both coupled with UB) sanitizers for
        # specific Bookworm configurations.
```

Small edit since sanitizers aren't enabled for all Bookworm distros.

Reply:
done